phpBB/includes/acp/acp_prune.php

```diff
@@ -327,7 +327,7 @@ function prune_users($id, $mode)
 						'USERNAME'			=> $usernames[$user_id],
 						'USER_ID'           => $user_id,
 						'U_PROFILE'			=> get_username_string('profile', $user_id, $usernames[$user_id]),
-						'U_USER_ADMIN'		=> ($auth->acl_get('a_user')) ? append_sid("{$phpbb_admin_path}index.$phpEx", 'i=users&amp;mode=overview&amp;u=' . $user_id, true, $user->session_id) : '',
+						'U_USER_ADMIN'		=> ($auth->acl_get('a_user')) ? append_sid("{$phpbb_admin_path}index.$phpEx", 'i=users&amp;mode=overview&amp;u=' . $user_id) : '',
```

Comment:
Do we still need `append_sid` at all?

Reply:
With the current change we'll still have the fallback to force the old behavior with the mentioned config setting. Maybe it could also be possible to move the cookie settings to a config file to allow fixing this manually of needed? I would however see this in an additional ticket and is IMHO outside the scope of this one.